tests/open_sgid/test_DatabaseConnection.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+A module that contains tests for the open_sgid.DatabaseConnection class
+"""
+
+from collections import namedtuple
+from typing import NamedTuple
+from unittest import mock
+
+from masquerade.providers.open_sgid import DatabaseConnection
+
+
+@mock.patch('masquerade.providers.open_sgid.psycopg2')
```

Comment:
**thought**: it might be nice to have a test where the mock has multiple return values where the connection is sometimes closed and sometimes open and you can call query n times and assert that the query was made on an open connection